src/element-templates/cmd/RemoveElementTemplateHandler.js

```diff
@@ -133,6 +137,33 @@ export default class RemoveElementTemplateHandler {
 
     return newBo;
   }
+
+  /**
+   * Copy documentation and execution listeners from source to target business object.
+   *
+   * NOTE: Mutetes the target business object.
+   * @param {ModdleElement} sourceBusinessObject
+   * @param {ModdleElement} targetBusinessObject
+   */
+  _copyNonTemplateProperties(sourceBusinessObject, targetBusinessObject) {
```

Comment:
Great solution, in the appropriate place, I think.

[Q]: This handles both C7 and C8. Is the behavior applicable for C7? If so, the standard way would be to split both implementations, and and move the cloud one to `cloud-element-templates`.

Reply:
As C7 and C8 templates work slightly differently I'd be fine to only implement this change for C8.

Reply:
`cloud-element-templates` actually [use the same remove handler](https://github.com/bpmn-io/bpmn-js-element-templates/blob/d60bbd267fc1296acf2862e6db071120a65ebbf4/src/cloud-element-templates/cmd/ElementTemplateCommands.js#L2).

Should we introduce a place for common code?

Reply:
Yes, `element-templates` is not the place for `zeebe` properties.

Reply:
I extracted some common code in https://github.com/bpmn-io/bpmn-js-element-templates/pull/120/commits/a7b14c4b9542d802a63166a174fc3dc0da92c65e